src/main/java/sally/parser/Parser.java

```diff
@@ -0,0 +1,134 @@
+package sally.parser;
+
+import sally.command.AddCommand;
+import sally.command.ByeCommand;
+import sally.command.Command;
+import sally.command.DeleteCommand;
+import sally.command.MarkCommand;
+import sally.command.UnmarkCommand;
+import sally.exception.SallyException;
+import sally.storage.Storage;
+import sally.task.*;
```

Comment:
Imported packages should be listed explicitly, maybe you could write down the attributes and methods you need from the `task` package and import in one-by-one? :) Otherwise LGTM!